tests/data/test_portfolio.py

```diff
@@ -220,17 +220,17 @@ async def test_update_portfolio_with_filled_orders(backtesting_trader):
 
     # when filling limit buy
     await fill_limit_or_stop_order(limit_buy)
-    assert portfolio_manager.portfolio.get_currency_portfolio("BTC", PORTFOLIO_AVAILABLE) == 8.999
+    assert portfolio_manager.portfolio.get_currency_portfolio("BTC", PORTFOLIO_AVAILABLE) == 9
```

Comment:
what happened here ? fees ?

Reply:
oh yes I forgot to add fee calculation :+1:

Reply:
Done :wink: :ok_hand:

Reply:
💯